wolfcrypt/src/ed448.c

```diff
@@ -329,7 +377,8 @@ int wc_ed448ph_sign_msg(const byte* in, word32 inLen, byte* out, word32 *outLen,
     int  ret = 0;
     byte hash[64];
 
-    ret = wc_Shake256Hash(in, inLen, hash, sizeof(hash));
+    ret = ed448_hash(key, in, inLen, hash, sizeof(hash));
```

Comment:
`ret` above does not need initialized... always set here. Some compilers will bark about set not used.

Reply:
Also there are two places where `[64]` is hard coded for the ed448 signature hash size. Can this be added to the ed448 header or at top of .c? From RFC8032 5.2 `Ed448ph is the same but with PH being SHAKE256(x, 64)`.

Reply:
done